core/server/master/src/test/java/alluxio/master/file/meta/InodeTreeTest.java

```diff
@@ -305,6 +306,28 @@ public void createPathTest() throws Exception {
     // file was created
     assertEquals(1, created.size());
     assertEquals("file", created.get(0).getName());
+
+    // Test inheritance
```

Comment:
Can we easily put this in a separate test method?

Reply:
yes